debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -462,6 +482,10 @@ public boolean hasUnchangedToastColumnMarker() {
             return true;
         }
 
+        public boolean sendsNullToastedValuesInOld() {
```

Comment:
Sigh, seeing the need for this method makes my eyes bleed... Could you also create an issue with wal2json, it seems it should be fixed there ideally?